VERSIONING.md

```diff
@@ -0,0 +1,220 @@
+# Versioning
+
+This document describes the versioning policy for this repository. This policy
+is designed so the following goals can be achieved.
+
+**Users are provided a codebase of value that is stable and secure.**
+
+## Policy
+
+* Versioning of this project will be idiomatic of a Go project using [Go
+  modules](https://github.com/golang/go/wiki/Modules).
+    * [Semantic import
+      versioning](https://github.com/golang/go/wiki/Modules#semantic-import-versioning)
+      will be used.
+        * Versions will comply with [semver
+          2.0](https://semver.org/spec/v2.0.0.html) with the following exceptions.
+            * New methods may be added to exported API interfaces. All exported
```

Comment:
Why do we want to carve out this exception? Are we saying we think it is OK to break existing code that implements out interfaces? I think we should aim not to do this.

Reply:
See https://github.com/open-telemetry/opentelemetry-collector/pull/3476/files#r656310438 for lineage.  I will remove this from the next draft as it seems not applicable to the collector.